src/encoded/static/components/layout.js

```diff
@@ -54,13 +54,12 @@ var BlockEditModal = React.createClass({
 
     render: function() {
         var blocktype = globals.blocks.lookup(this.props.value);
-        var schema = blocktype.schema();
         var BlockEdit = blocktype.edit || FallbackBlockEdit;
         return (
-            <Modal actuator={this.props.actuator}>
+            <Modal ref="modal" actuator={this.props.actuator}>
```

Comment:
I don't think this a problem with the Modal component. It's a feature of the BlockEditModal, to automatically open the settings modal for a new block as soon as it is dragged onto the layout.